Documentation/articles/getting_started/1_setting_up_your_development_environment_windows.md

```diff
@@ -2,35 +2,64 @@
 
 This section provides a step-by-step guide for setting up your development environment on Windows.
 
-MonoGame can work with most .NET compatible tools, but we recommend [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) (prior versions are not supported with MonoGame 3.8.1).
+MonoGame can work with most .NET compatible tools, but we recommend [Visual Studio 2022](https://visualstudio.microsoft.com/vs/)
+
+> [!IMPORTANT]
+> Prior version of Visual Studio are not supported with MonoGame 3.8.1
 
 Alternatively, you can use [JetBrains Rider](https://www.jetbrains.com/rider/) or [Visual Studio Code](https://code.visualstudio.com/).
 
 ## [Recommended] Install Visual Studio 2022
 
-Before using MonoGame, you will need to install [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) or later (any edition, including Community) with the following workloads, depending on your desired [target platform(s)](~/platforms.md):
+Before using MonoGame, you will need to install [Visual Studio 2022](https://visualstudio.microsoft.com/vs/)
+
+> [!NOTE]
+> You can install any edition of Visual Studio, including the Community edition.
```

Comment:
I'd say this is redundant.

Reply:
I agree, this information is redundant and the alert could be removed.  I just took the existing verbiage and moved it into an alert box.

Reply:
Removed redundant information in alert box.